torchstore/transport/pipe.py

```diff
@@ -131,12 +136,16 @@ class Pipe:
     Transport wrapper for communicating from local clients to storage volumes.
     """
 
-    def __init__(self, storage_volume) -> None:
-        self.storage_volume = storage_volume
+    def __init__(self, storage_volume_ref: "StorageVolumeRef") -> None:
+        self.storage_volume_ref = storage_volume_ref
+        self.transport_context = storage_volume_ref.transport_context
+        self.storage_volume = storage_volume_ref.volume
 
     def create_transport_buffer(self) -> TransportBuffer:
         # TODO: eventually this should be dependent on the connections available to a storage_volume
-        if rdma_available():
+        if torchcomms_rdma_available():
```

Comment:
same comment as earlier, I think this can be done away with default transport type